src/content/posts/2022/bootstrap-4-6-2.md

```diff
@@ -0,0 +1,37 @@
+---
+author: mdo
+date: "2022-07-14T21:30:00Z"
+title: Bootstrap v4.6.2
+video: RDTzFnYcIqj6I
```

Comment:
There is an issue with the video which doesn't seem to exist (see https://81222cdd.blog-5aq.pages.dev/ and https://www.youtube.com/watch?v=RDTzFnYcIqj)
```
An error occurred. Please try again later. (Playback ID: B7O6aNdDb6d3_cFp)
```